src/NuGet.Clients/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -424,6 +444,20 @@ private async Task<IReadOnlyList<RestoreSummary>> PerformNuGetV2RestoreAsync(Pac
             }
         }
 
+        private static Dictionary<string, string> GetPackagesConfigToProjectPath(PackageRestoreInputs packageRestoreInputs)
+        {
+            Dictionary<string, string> configToProjectPath = [];
```

Comment:
What is this syntax?  I prefer 
```suggestion
            Dictionary<string, string> configToProjectPath = new();
```

I couldn't find any instance of this syntax in our existing code and suggest we don't start using it.

Reply:
These are largely the suggestions applied when you change the constructor. 
I think it used to do `new()` but now it does `[]`. 

I can change it.

Reply:
It confused me, I thought it was somehow set to an empty array?  But it must be short hand for `new()`?  I've never seen it and think `new()` is short hand enough while still being concise.